packages/shared/styles/components/atoms/SfQuantitySelector.scss

```diff
@@ -1,22 +1,71 @@
 @import "../../helpers";
 
 .sf-quantity-selector {
-  --input-width: 1.875rem;
-  --input-height: 1.6875rem;
-  input {
-    --input-padding: 0;
-    --input-margin: 0;
-    --input-border-width: 1px;
-    text-align: center;
-    transition: border-color 150ms linear;
-    &:focus {
-      --input-border: 1px solid var(--c-primary);
-      --input-border-width: 1px 1px 0 1px;
-      --input-padding: 0 0 1px 0;
+  display: flex;
```

Comment:
```suggestion
  box-sizing: border-box;
  display: flex;
```

Reply:
when you set width and height you must remember on change `box-sizing` to `border-box`